### ISSUE TITLE
Limit Request: h2o-pysparkling-3.1 - 200 MB

### ISSUE THREAD

**Issue by mn-mikke** [pypa/pypi-support#951]
### Project

h2o-pysparkling-3.1

### Does this project already exist?

- [X] Yes

### Size of release/project

200 MB

### Which indexes

PyPI, TestPyPI

### Reasons for the request

h2o-pysparkling-3.1 is a new project representing a pysparkling build for Apache Spark 3.1. The size of Pysparkling artifacts are over the limit 100MB.  I'm kindly ask you to change the limit to 200MB as it's already done for other pysparkling projects (builds) h2o-pysparkling-x.y (where x.y could is 2.1, 2.2, 2.3, 2.3, 3.0).

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

**Comment by mn-mikke** [comment]
Hello, any update on this issue? The release of pysparkling support for Apache Spark 3.1 is currently blocked by not being able to publish artifacts to PyPi.

Thanks a lot for your help!

**Comment by mn-mikke** [comment]
cc @di

**Comment by di** [comment]
I've set the upload limit for `h2o-pysparkling-3.1` to 200 MB on PyPI. Please be mindful of the frequency of releases at that size.

This project does not exist on TestPyPI, if you want me to increase the limit there, you'll need to make a release under the limit first.

**Comment by mn-mikke** [comment]
@di Thank you very much!
